pkg/controller/common.go

```diff
@@ -565,8 +565,11 @@ func (c *SharedState) addPod(ctx context.Context, pod *v1.Pod, updateOption upda
 		}
 		namespacedClaimName := getQualifiedName(pod.Namespace, volume.PersistentVolumeClaim.ClaimName)
 		if _, ok := c.claimToVolumeMap.Load(namespacedClaimName); !ok {
-			w.Logger().V(5).Infof("Skipping Pod %s. Volume %s not csi. Driver: %+v", pod.Name, volume.Name, volume.CSI)
-			continue
+			// Log message if the Pod status is not Running
+			if pod.Status.Phase == v1.PodRunning {
+				w.Logger().V(5).Infof("Skipping Pod %s. Volume %s not csi. Driver: %+v", pod.Name, volume.Name, volume.CSI)
+				continue
```

Comment:
We would need this `continue` to be outside of `if pod.Status.Phase == v1.PodRunning`.

Reply:
Updated